packages/fxa-settings/src/models/experiments/index.ts

```diff
@@ -0,0 +1 @@
+export * from './key-stretch-experiment';
```

Comment:
Not sure if this needs a copyright header.

```suggestion
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

export * from './key-stretch-experiment';
```